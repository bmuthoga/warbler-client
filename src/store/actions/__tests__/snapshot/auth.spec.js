import configureMockStore from 'redux-mock-store'
import { setCurrentUser } from '../../auth'

describe('Auth Action Creators Snapshot Tests', () => {
  let store
  const mockStore = configureMockStore()

  beforeEach(() => {
    store = mockStore({})
  })

  it('should create SET_CURRENT_USER action if signup/signin is successful', () => {
    const user = {
      id: '7784fh489g90239f45',
      username: 'testUser',
      profileImageUrl: 'https://lorempixel.com/200/200/'
    }
    store.dispatch(setCurrentUser(user))
    expect(store.getActions()).toMatchSnapshot()
  })  
})
