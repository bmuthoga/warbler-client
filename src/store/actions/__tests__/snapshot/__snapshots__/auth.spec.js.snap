// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Action Creators Snapshot Tests should create SET_CURRENT_USER action if signup/signin is successful 1`] = `
Array [
  Object {
    "type": "SET_CURRENT_USER",
    "user": Object {
      "id": "7784fh489g90239f45",
      "profileImageUrl": "https://lorempixel.com/200/200/",
      "username": "testUser",
    },
  },
]
`;
